common/changes/@microsoft/rush/nickpape-lock-home_2018-02-15-21-09.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Add locking when installing the package manager tool in the user's home folder.",
```

Comment:
>Add [](start = 18, length = 3)

"Fix an issue where folders could get corrupted if the Rush tool was accidentally invoked multiple times concurrently in the same folder"